src/test/java/com/hocs/test/glue/decs/ValidationStepDefs.java

```diff
@@ -1171,96 +1171,96 @@ public void iTriggerTheErrorMessageAtTheStage(String errorMessage, String stage)
                                 safeClickOn(continueButton);
                                 break;
                             case "REASON FOR TRANSFER REQUIRED":
```

Comment:
for all of these steps you have directly referenced WebElements from the COMP pages. Why arent you using the methods from those pages? Then we can alter just the mehtods if the page changes, and easily find all the references to those elements by seeing where the methods are used.
I know in some cases the mothods might include too many actions, and you have to directly reference a page object, but I think ideally we should keep that to a minimum.
Having 2 ways that a WebElement is referenced in the pack means we have to do more work in finding how a change effects the scenarios.